### PR TITLE
Add PNaCl target info to the makefile target cfgs and initialize the PNaCl target machine if available.

### PR DESCRIPTION
**Pull request by DiamondLovesYou**
r? @alexcrichton

### CODE REVIEW

File: src/librustc_llvm/lib.rs [hunk 1]
```diff
@@ -2357,6 +2360,20 @@ pub fn initialize_available_targets() {
     init_target!(have_component_aarch64 AArch64);
     init_target!(have_component_arm ARM);
     init_target!(have_component_x86 X86);
+
+    // PNaCl doesn't provide some of the optional target components, so we
+    // manually initialize it here.
+    #[cfg(have_component_pnacl)]
+    fn init() {
```
Comment (alexcrichton) [inline src/librustc_llvm/lib.rs]: Could this be `init_pnacl` b/c it's pnacl specific?

**Comment by alexcrichton** [comment]
just one small nit but otherwise r=me

**Comment by DiamondLovesYou** [comment]
Done.

**Comment by DiamondLovesYou** [comment]
Fixed.

**Pull request merged**